### PR TITLE
Edit: account management

### PR DESCRIPTION
**Pull request by Mocab**

### CODE REVIEW

File: _posts/falix/general/2021-08-04-account-management.md [hunk 1]
```diff
@@ -16,11 +16,8 @@ It is currently not possible to transfer your Falix server and account to anothe
 
 # Managing
 ## Resetting Your Password
-If you've forgotten your account's password for your Game Panel, please click on Forgot Password button, which should be under the Login button. Type in your Game Panel email and wait for an email to arrive in your inbox.
-
-You could also log into the Client Panel and head over to your profile settings, then click on reset Game Panel password, which will automatically generate a random password for you to use.
-
-If you've not forgotten your password and just want to use a new password, click on the account icon at the top of the Game Panel, then reset your password there.
+If you've forgotten your account's password for your Game Panel, please click on your profile picture in the [client page](client.falixnodes.net/) then click on "[settings](https://client.falixnodes.net/profile/settings)", it should be at the top right corner of your screen. You should be redirected to another page. Scroll down and click on "Change Game Panel Password". You will be redirected back to the main page, scroll down and you should see an alert containing your password.
```
Comment (itsjuoum) [inline _posts/falix/general/2021-08-04-account-management.md:19]:
```suggestion
If you've forgotten your account's password for your Game Panel, please click on your profile picture in the [client page](client.falixnodes.net/) then click on "[Settings](https://client.falixnodes.net/profile/settings)", it should be at the top right corner of your screen. You should be redirected to another page. Scroll down and click on "Change Game Panel Password". You will be redirected back to the main page, scroll down and you should see an alert containing your password.
```

File: _posts/falix/general/2021-08-04-account-management.md [hunk 2]
```diff
@@ -16,11 +16,8 @@ It is currently not possible to transfer your Falix server and account to anothe
...
+![image](../../../assets/images/posts/falix/account-management/1.png)
```
Comment (itsjuoum) [inline _posts/falix/general/2021-08-04-account-management.md:20]: Naming an image "1" is not a good practice. The file name should be about what the image is. Something like "password-alert-box.png"

**Review by itsjuoum — CHANGES_REQUESTED** [review]

File: _posts/falix/general/2021-08-04-account-management.md [hunk 3]
```diff
@@ -16,11 +16,8 @@ It is currently not possible to transfer your Falix server and account to anothe
 
 # Managing
 ## Resetting Your Password
-If you've forgotten your account's password for your Game Panel, please click on Forgot Password button, which should be under the Login button. Type in your Game Panel email and wait for an email to arrive in your inbox.
-
-You could also log into the Client Panel and head over to your profile settings, then click on reset Game Panel password, which will automatically generate a random password for you to use.
-
-If you've not forgotten your password and just want to use a new password, click on the account icon at the top of the Game Panel, then reset your password there.
+If you've forgotten your account's password for your Game Panel, please click on your profile picture in the [Client Page](client.falixnodes.net/) then click on "[Settings](https://client.falixnodes.net/profile/settings)", it should be at the top right corner of your screen. You should be redirected to another page. Scroll down and click on "Change Game Panel Password". You will be redirected back to the main page, scroll down and you should see an alert containing your password.
```
Comment (DaddyChimken) [inline _posts/falix/general/2021-08-04-account-management.md:19]:
```suggestion
If you've forgotten your account's password for the Game Panel, click on your profile picture in the top right corner of your screen on the [Client Page](client.falixnodes.net/), then click on "[Settings](https://client.falixnodes.net/profile/settings)". You should be redirected to another page. 
Scroll down and click on "Change Game Panel Password". You will be redirected back to the main page, scroll down and you should see a success alert containing your password.
```

**Review by DaddyChimken — CHANGES_REQUESTED** [review]

File: _posts/falix/general/2021-08-04-account-management.md [hunk 4]
```diff
@@ -16,11 +16,9 @@ It is currently not possible to transfer your Falix server and account to anothe
 
 # Managing
 ## Resetting Your Password
-If you've forgotten your account's password for your Game Panel, please click on Forgot Password button, which should be under the Login button. Type in your Game Panel email and wait for an email to arrive in your inbox.
-
-You could also log into the Client Panel and head over to your profile settings, then click on reset Game Panel password, which will automatically generate a random password for you to use.
-
-If you've not forgotten your password and just want to use a new password, click on the account icon at the top of the Game Panel, then reset your password there.
+If you've forgotten your account's password for the Game Panel, click on your profile picture in the top right corner of your screen on the [Client Page](client.falixnodes.net/), then click on "[Settings](https://client.falixnodes.net/profile/settings)". You should be redirected to another page.
```
Comment (DaddyChimken) [inline _posts/falix/general/2021-08-04-account-management.md:19]: The client & panel site are both panels. Please remember this.

```suggestion
If you've forgotten your account's password for the Game Panel, click on your profile picture in the top right corner of your screen on the [Client Panel](client.falixnodes.net/), then click on "[Settings](https://client.falixnodes.net/profile/settings)". You should be redirected to another page. 
```

**Review by DaddyChimken — CHANGES_REQUESTED** [review]

**Review by DaddyChimken — APPROVED** [review]

**Pull request merged**